site/source/docs/getting_started/Tutorial.rst

```diff
@@ -87,13 +87,13 @@ execute it. You can run them using :term:`node.js`:
 This prints "hello, world!" to the console, as expected.
 
 .. note:: Older node.js versions do not have WebAssembly support yet. In that
-   case you will see an error message suggesting that you build with
+   case, you will see an error message suggesting that you build with
    ``-sWASM=0`` to disable WebAssembly, and then emscripten will emit the compiled
    code as JavaScript. In general, WebAssembly is recommended as it has
    widespread browser support and is more efficient both to execute and to
    download (and therefore emscripten emits it by default), but sometimes you
-   may need your code to run in an environment where it is not yet present and
-   so should disable it.
+   may need your code to run in an environment where it is not yet present,
+   so you should disable it.
```

Comment:
I think we can probably just delete this whole note.   Versions of node that don't support WebAssembly are ancient at this point.